tests/test_client.py

```diff
@@ -26,46 +25,32 @@ def test_init(api_client: ApiClient):
     assert api_client
 
 
-def test_timeout_default(httpx_mock: HTTPXMock):
-    httpx_mock.add_response(
-        match_extensions={
-            "timeout": {
-                "connect": DEFAULT_TIMEOUT_CONFIG.connect,
-                "read": DEFAULT_TIMEOUT_CONFIG.read,
-                "write": DEFAULT_TIMEOUT_CONFIG.write,
-                "pool": DEFAULT_TIMEOUT_CONFIG.pool,
-            }
-        },
-        status_code=httpx.codes.BAD_GATEWAY,
+def test_base_url_override(respx_mock: respx.mock):
+    respx_mock.get(re.compile(r"^https://test/repos/foo/bar/import/issues")).mock(
+        return_value=httpx.Response(
+            httpx.codes.OK, text=get_fixture("response-multiple-check-status-of-multiple-issues.json")
+        )
+    )
+    ApiClient(http_client=HttpClient(token=GITHUB_TOKEN, base_url="https://test")).get_status_multiple(
```

Comment:
In theory, you can write this test differently by saving `respx_mock.get(...).mock(...)` in a variable that gives access to request object. But in this case you also should instantiate `respx_mock` without specifying url to catch all possible routes.

Reply:
Good point, I thought as much... but my impression is that "good style" is to describe things to be matches as narrowly as possible and rely on uncaught matches rather than trying to check the request for properties.

This is how I rewrote the "big" `test_import_issue` test. I just specify what has to match as closely as possible and remove assertions on the request. Seems like a good idea indeed.